data/templates/install-ubuntu/ubuntu-preseed

```diff
@@ -34,7 +34,13 @@ d-i mirror/http/hostname string <%=repoHostname%>
 # Get directory from repo
 <% var repoDirectory = repo.replace(/^(https?:\/\/)?([a-zA-Z0-9\.:-]+)/g,"") -%>
 d-i mirror/http/directory string <%=repoDirectory%>
-d-i preseed/early_command string umount /media || true
+
+d-i preseed/early_command string \
+<%_ if( typeof progressMilestones !== 'undefined' && progressMilestones.preConfigUri ) { _%>
+    wget 'http://<%=server%>:<%=port%><%-progressMilestones.preConfigUri%>' || true; \
+<%_ } _%>
+    umount /media || true;
```

Comment:
Just as you've mentioned the whole ubuntu-preseed is not indent. I just mean keep consistent, even it exists previously, correct it if you think it's better.